assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -159,6 +159,7 @@ function Title({
           handleHeightChange={handleHeightChange}
           handleExpandToHeightChange={handleExpandToHeightChange}
           handleDoubleClick={resetHeight}
+          position="top"
```

Comment:
Any reason to add this if `top` is the default anyway?

Reply:
No, can probably remove it.